src/sass/_BrandIcon.scss

```diff
@@ -1,19 +1,108 @@
 // Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
 
-@include ms-brand-icon-classes(1);
+@include ms-brand-icon-classes('1');
 @include ms-brand-icon-size-classes();
 
 // Pixel Ratio 1.5
 @media only screen and (min-resolution: 144dpi) {
-  @include ms-brand-icon-classes(1_5);
+  @include ms-brand-icon-classes('1_5');
 }
 
 // Pixel Ratio 2
 @media only screen and (min-resolution: 192dpi)  {
-  @include ms-brand-icon-classes(2);
+  @include ms-brand-icon-classes('2');
 }
 
 // Pixel Ratio 3
 @media only screen and (min-resolution: 288dpi) {
-  @include ms-brand-icon-classes(3);
-}
\ No newline at end of file
+  @include ms-brand-icon-classes('3');
+}
+
+// The 'onepkg' and 'xls' icons have been deprecated. These classes will be removed in
+// a future release. Until then, they are mapped to 'one' and 'xlsx'.
+
+.ms-BrandIcon--onepkg.ms-BrandIcon--icon16 {
+  background-image: url(https://spoprod-a.akamaihd.net/files/fabric/assets/item-types/16/one.png);
```

Comment:
nit: the protocol/domain part of the URL could be pulled into something like a `baseUrl` variable to minimize repetition.

Reply:
I'll leave it as is, since there shouldn't be any changes needed here before we remove it entirely.